docs/changelog.rst

```diff
@@ -1,5 +1,46 @@
 Changelog
 ================================
+CBC SDK 1.4.0 - Released TBD
+----------------------------
+
+**Breaking Changes:**
+
+* ``Policy`` object has been moved from ``cbc_sdk.endpoint_standard`` to ``cbc_sdk.platform``, as it now uses the new
```

Comment:
Maybe break this out to sub-points and add some direction on what customers should do:

* ``Policy`` object has been moved from ``cbc_sdk.endpoint_standard`` to ``cbc_sdk.platform`` and now uses the new
  Policy Services API rather than the old APIs through Integration Services.
  
  * **N.B.:** This change means that you *must* use a custom API key with permissions under ``org.policies`` to manage policies, rather than an older "API key."

  * To enable time to update integration logic the ``cbc_sdk.endpoint_standard Policy`` object may still be imported from the old package, and supports operations that are backwards-compatible with the old one.

  * When developing a new integration, or updating an existing one ``cbc_sdk.platform`` should be used.  There is a utility class ```Policy Builder``` and as features are added to Carbon Black Cloud they will be added to this module.

Reply:
I set it up just as you have it, with only a few formatting changes.